ObjCVoiceCallKitQuickstart/ViewController.m

```diff
@@ -181,6 +182,7 @@ - (void)callDidConnect:(TVOCall *)call {
     NSLog(@"callDidConnect:");
 
     self.call = call;
+    self.callKitCompletionCallback(YES);
```

Comment:
Minor - you could also nil `self.callKitCompletionCallback` after calling it here, and in the failure case.